about.html

```diff
@@ -0,0 +1,155 @@
+        </ul>
+      </div>
+    </nav>
+
+    <div class="burger" id="burger">
+      <div class="line"></div>
+      <div class="line"></div>
+      <div class="line"></div>
+    </div>
+    <!--Section Hero-->
+    
+      <section class="introduction" id="home">
+
+        <h1 class="about-second-heading">THE BIGGEST<br /> FASHION EVENT 2022</h1>
+
+        <div class="about-para-border">
+          <p>
+            The DC Fashion Show is a conference brings fashion to come together from all over the world in order to
+            share their crazy ideas and talk about trendigs, those discussions can be more like in our own small nation,
+            but also in terms of fashion.
+            <span class="highlight">fashion </span> world
+          </p>
+        </div>
+
+        <div class="about-contact">
+          <p>
+            Please contact us per Email for any further questions about DC Fashion
+            Show!<br /><strong><u>dcfashionshow@ccomunity.org</u></strong>
+          </p>
+        </div>
+      </section>
+    </header>
+    <section class="create-common">
+      <h1>Creative Designs Soft-Stiches 2022</h1>
+      <br />
+      <br />
+      <p>
+        The name Soft-Stiches came from the magic hands on a fabric. A stitch
+        with the touch of magic..
+      </p>
+      <br />
+      <br />
+
+      <img class="card-logo" src="./images/text-1653551772639.jpg" alt="Mirela Pitt's picture">
+    </section>
+
+
+    <section class="past-reviews">
+      <h1>See the past Soft-Stiches Global Conference</h1>
+      <hr>
+      <p>
+        The two previous Soft-Stiches Global conference held in France 2016
+        and Tokoyo 2019
+      </p>
+
+      <div class="card-preview">
+        <div class="card-one">
+          <div class="cardBg"></div>
+          <h1 class="index">2016</h1>
+          <p>Soft-Stiches in France 2016</p>
+        </div>
+        <div class="card-two">
+          <div class="cardBg"></div>
+          <h1 class="index">2019</h1>
+          <p>Soft-Stiches in Tokoyo 2019</p>
+        </div>
+      </div>
+    </section>
+    </main>
+    <!-- Footer -->
+
+    <footer>
+      <div class="foot foot-about">
+        <div class="foot-heading">
+          <h1>Partner</h1>
+          <hr>
+        </div>
+        <div class="partners">
+          <div id="Stitch-up">STITCH UP</div>
+          <div id="Air-ups">Hope Fashion</div>
+          <div id="Mozzi-ups">Hot Make-up</div>
+          <div id="Air-up">Air UP</div>
+          <div id="Mozzi-up">Mozzi UP</div>
+        </div>
+      </div>
+
+      <section class="close-foot-aboutcolor">
+        <div class="about-close-foot">
+          <div class="logo">
+            <h3>Soft-Stiches</h3>
+          </div>
+          <div class="rights">
+            <h4>2022 Some Right Reserved</h4>
+            <p>Proundly sponsored by<span class="talic">Soft-Stiches</span></p>
+          </div>
+        </div>
+      </section>
+    </footer>
+
```

Comment:
- [ ] Great job so far 👏🏻  However your footer section has two-child, a `logo` aligned to the left and `text` aligned to the right. It would be great if you could add that too 




- [ ] Change the text color of the Home and About Page, you have to stick to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), your's it's almost centering, kindly fix this.

- Your design:

![fot42](https://user-images.githubusercontent.com/79058364/170615850-36fe354a-1dba-4101-9f91-916986d06fd6.png)



- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![footabou1](https://user-images.githubusercontent.com/79058364/170037105-cdec3174-540d-42d9-96a6-5390be014cae.png)